j2eeapis/external/deployment-api-1.2-rev-1-license.txt

```diff
@@ -0,0 +1,832 @@
+Name: Java EE Deployment API
+License: CDDL-GPL-2-CP
+Description: Java EE Deployment API
+Version: 1.2-rev-1
```

Comment:
The jar has CDDL-GPL-2-CP license.
On run of "ant -Dcluster.config=full verify-libs-and-licenses" I am getting below error.
"j2eeapis/external/deployment-api-1.2-rev-1-license.txt has a GPL-family license but is either not covered by the Classpath Exception, or is not compile-time/optional only"

In some instances it is used as runtime dependency, like
./apisupport/apisupport.ant/test/unit/src/org/netbeans/modules/apisupport/project/ProjectXMLManagerTest.java: "ext/deployment-api-1.2-rev-1.jar\n" +
./j2eeapis/nbproject/project.xml: ext/deployment-api-1.2-rev-1.jar

So is it okay to use the jar, if so then should I ignore the above license validation message or I need to add "type: compile-time" as license header in license file
Please advice